lib/__tests__/_sendEvent.test.ts

```diff
@@ -41,17 +43,21 @@ describe("sendEvent", () => {
     });
     it("should pass over the payload with multiple events", () => {
       (AlgoliaInsights as any).sendEvent("click", {
-        objectID: "1"
+        eventName: "my-event",
+        indexName: "my-index",
+        objectID: "1",
+        position: 3
       });
       expect(XMLHttpRequest.send).toHaveBeenCalledTimes(1);
       const payload = JSON.parse(XMLHttpRequest.send.mock.calls[0][0]);
       expect(payload).toEqual({
         events: [
           expect.objectContaining({
             eventType: "click",
-            objectID: "1",
-            userID: "42",
-            timestamp: expect.any(Number)
```

Comment:
timestamp is not provided anymore?

Reply:
it is but I'm purposely not checking that here.
I wanted to keep this about the payload passed to `XMLHttpRequest.send` so I added only required parameters.
This is why i've removed userID and timestamp

To be fully consistent, I'll also remove objectID and positions (as they are also optional)

Reply:
It's done. It should be more minimal now.